mslib/msui/msui_mainwindow.py

```diff
@@ -822,7 +825,7 @@ def create_new_flight_track(self, template=None, filename=None, function=None, a
             # function is none if ftml file is selected
             if function is None:
                 try:
-                    waypoints_model = ft.WaypointsTableModel(filename=str(filename))
+                    waypoints_model = ft.WaypointsTableModel(filename=filename)
```

Comment:
hmm, this makes an os.path string from the Path, was there a problem solved by the change, which one?